docs/graph-data-init-container.md

```diff
@@ -0,0 +1,48 @@
+# Graph Data Init Container
+
+Normally, the graph data is fetched directly from the Cincinnati graph
+data repository at: http://github.com/openshift/cincinnati-graph-data.
```

Comment:
nit: http -> https  Also might want to make this a formal link so readers don't have to think about Markdown link detection vs. trailing periods.